File: beecrowd_exercicios_em_Ruby/strings/recharged_unlucky_numbers.rb
def chk(n)
  for i in (0..n.length-1)
    if n[i] == '1' && n[i+1] == '3'
      puts "#{n} es de Mala Suerte"
      return
    end
  end

  puts "#{n} NO es de Mala Suerte"

end

number = gets.chomp.to_s

chk(number)

File: beecrowd_exercicios_em_Python/string/recharged_unlucky_numbers.py
def chk(n):
    for i in range(len(n) - 1):
        if n[i] == "1" and n[i + 1] == "3":
            print(n,"es de Mala Suerte")
            return

    print(n,"NO es de Mala Suerte")


n = input()

chk(n)

